includes/class/heygov-resource.php

```diff
@@ -49,5 +50,44 @@ public function load_apps_banner() {
 			<?php
 		}
 	}
+	function heygov_forms_shortcode( $atts = array()) {
+		$args = shortcode_atts( array(
+			'container-small' => false,
+			'department' => 'all'
+		), $atts );
+
+		$small = $args['container-small']; 
+		$department = $args['department'];
+
+        ob_start(); 
+        // HeyGov ID
+        $heygov_id = get_option('heygov_id');
+
+       // Get any existing copy of our transient data
+        if ( false === ( $forms = get_transient( 'forms' ) ) ) {
+            // It wasn't there, so regenerate the data and save the transient
+			$forms = wp_remote_get('https://heygov-api-develop-nxb3467cgq-uc.a.run.app/'.$heygov_id.'/forms?status=public&expand=department&department='.$department);
+				if (is_wp_error($forms)) {
+					return $forms;
+				}
+				$forms = wp_remote_retrieve_body($forms);
+				$forms = json_decode($forms);
+            set_transient( 'forms', $forms, 12 * HOUR_IN_SECONDS );
+       }
+	   
+        require_once HEYGOV_DIR . 'includes/view/show-heygov-muni-forms.php';
+		if($small === "true" ) {
```

Comment:
this is something that shouldn't be in final version 😕 
instead of `container-small` attribute it should be `container` or `maxcolumns` that has the value or "small", "medium", etc and the actual rows should be handled with CSS, not JS

Reply:
Hey Andrei, JS only add & remove custom class, so it is handled by css.